Q/QuantLib/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "QuantLib"
+version = v"1.31"
+
+sources = [
+    GitSource("https://github.com/lballabio/QuantLib.git", "38551644cb8b9b6b794f443225e522296ce08331"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/QuantLib
+install_license LICENSE.TXT
+mkdir build
+cd build
+if [[ "${target}" == *-linux-* ]]; then
```

Comment:
This is needed only for x86 glibc
```suggestion
if [[ "${target}" == *86*-linux-gnu ]]; then
```